### PR TITLE
feat: remove proxied wallets

### PR DESCRIPTION
**Pull request by Asmadek**
Receive proxiedes accounts from proxy service which was deleted. Remove it with related wallets and balances.

### CODE REVIEW

**Comment by Asmadek** [comment]
Task linked: [CU-86933z3r7 Proxy Service: Handle proxied account removal: Remove proxied wallet](https://app.clickup.com/t/86933z3r7)

File: src/renderer/entities/wallet/model/wallet-model.ts [hunk 1]
```diff
@@ -161,6 +162,10 @@ const removeWalletFx = createEffect(async ({ walletId, accountIds }: RemoveParam
   return walletId;
 });
 
+const removeWalletsFx = createEffect(async (wallets: RemoveParams[]): Promise<ID[]> => {
```
Comment (tuul-wq) [inline src/renderer/entities/wallet/model/wallet-model.ts:165]:
```suggestion
const removeWalletsFx = createEffect((wallets: RemoveParams[]): Promise<ID[]> => {
```

File: src/renderer/features/proxies/model/proxies-model.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ import { isDisabled, networkModel } from '@entities/network';
 import { proxyWorkerUtils } from '../lib/utils';
 import { accountUtils, walletModel } from '@entities/wallet';
 import { proxyModel, proxyUtils } from '@entities/proxy';
+import { balanceModel } from '@/src/renderer/entities/balance';
```
Comment (tuul-wq) [inline src/renderer/features/proxies/model/proxies-model.ts:25]:
```suggestion
import { balanceModel } from '@entities/balance';
```

File: src/renderer/features/proxies/workers/proxy-worker.ts [hunk 3]
```diff
@@ -161,7 +161,7 @@ async function getProxies(
   }
 
   const proxiesToRemove = proxies.filter((p) => existingProxies.some((ep) => isEqual(p, ep)));
-  const proxiedesToRemove = Object.values(accounts)
+  const proxiedesToRemove = Object.values(proxiedes)
```
Comment (tuul-wq) [inline src/renderer/features/proxies/workers/proxy-worker.ts:164]: Maybe `proxides`?

**Review by tuul-wq — COMMENTED** [review]
A bunch of questions

File: src/renderer/features/proxies/workers/proxy-worker.ts (hunk 3 above)
Comment (Asmadek) [reply src/renderer/features/proxies/workers/proxy-worker.ts:164]: we use `proxied` in [docs](https://docs.google.com/document/d/1xwzz-geiEvTxIPRPeAh_5R3PENuYjefzE5C-sRtaVtY/edit#heading=h.gfxjqyl4u0ai), looks like multiple should be `proxiedes`

**Review by egor0798 — APPROVED** [review]

**Pull request merged**